Solutions/Menes.Abstractions/Menes/Internal/OpenApiResponsesExtensions.cs

```diff
@@ -32,7 +32,7 @@ public static bool TryGetResponseForStatusCode(this OpenApiResponses responses,
         {
             if (statusCode.HasValue)
             {
-                if (responses.TryGetValue(statusCode.ToString(), out response))
+                if (responses.TryGetValue(statusCode.ToString()!, out response))
```

Comment:
The placement of that `!` looks odd. It's `statusCode` that is potentially nullable, so you'd expect this to be `statusCode?.ToString()!`.

But the more idiomatically-null-aware way to write this pair of `if` statements is:

```cs
if (statusCode is int statusCodeValue)
{
    if (responses.TryGetValue(statusCodeValue.ToString(), out response))
```

We're pattern matching to discover whether `statusCode` is null, and if it's not null, this puts the value into a definitely-not-null `int` variable that you can then use without any `!` or `?` fiddling about.